AsyncDisplayKitTests/ASXCTExtensions.h

```diff
@@ -0,0 +1,37 @@
+//
+//  ASDisplayNodeLayoutTests.m
+//  AsyncDisplayKit
+//
+//  Copyright (c) 2014-present, Facebook, Inc.  All rights reserved.
+//  This source code is licensed under the BSD-style license found in the
+//  LICENSE file in the root directory of this source tree. An additional grant
+//  of patent rights can be found in the PATENTS file in the same directory.
+//
```

Comment:
@Adlai-Holler Here we go #2158 unfortunately I don't have access to direct upstream master at the moment